src/components/List_Products.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react';
+import { saveItem } from '../lib/api';
+import { db } from '../lib/firebase';
+import { collection, onSnapshot } from 'firebase/firestore';
+
+function ListProducts() {
+  const [items, setItems] = useState(null);
+  const [itemName, setItemName] = useState('');
+
+  /* Get items */
+  useEffect(() => {
+    const unsubscribe = onSnapshot(collection(db, 'Products'), (snapshot) => {
+      let products = [];
+
+      snapshot.forEach((doc) => {
+        products.push({ ...doc.data(), id: doc.id });
+      });
+
+      console.log(products);
+      setItems(products);
+
+      return () => {
+        unsubscribe();
+      };
+    });
+  }, []);
+
+  /* Save Item */
+  const saveItemName = async (e) => {
+    e.preventDefault();
+    saveItem(itemName);
+    setItemName('');
+  };
+
+  return (
+    <div>
+      <form onSubmit={saveItemName}>
+        <input
+          value={itemName}
+          type="text"
+          onChange={(e) => setItemName(e.target.value)}
+          placeholder="Type your product..."
+        />
+        <button type="submit">Save</button>
+      </form>
+      {items &&
```

Comment:
if you change `const [items, setItems] = useState([]);` to have an array as an initial value there is no need for this condition `items &&`. what do y'all think about the initial value?